### DISCUSSION TITLE
How can I enable source maps for production?

### DISCUSSION THREAD

**Discussion by nullhook** [vercel/next.js#17817] [Help] [answered]
Is there an option where I can enable source maps for production?

I'm currently facing an issue on the production and I can't figure out it's original source.

**Comment by jamesmosier** [comment, marked as answer]
You can use [next-source-maps](https://github.com/vercel/next-plugins/tree/master/packages/next-source-maps) and then set whatever [webpack devtool](https://webpack.js.org/configuration/devtool/) you want to use to generate your source maps.
